pytorch_lightning/tuner/lr_finder.py

```diff
@@ -104,19 +90,21 @@ def lr_find(
 
         num_training: number of learning rates to test
 
-        mode: search strategy, either 'linear' or 'exponential'. If set to
-            'linear' the learning rate will be searched by linearly increasing
-            after each batch. If set to 'exponential', will increase learning
-            rate exponentially.
+        mode: Search strategy to update learning rate after each batch:
+
+            - ``'exponential'`` [default]: Will increase learning rate exponentially.
```

Comment:
If i'm not mistaken, parentheses are generally used for this

```suggestion
            - ``'exponential'`` (default): Will increase the learning rate exponentially.
```